dll/cpl/access/lang/pl-PL.rc

```diff
@@ -248,7 +248,7 @@ BEGIN
     GROUPBOX "Klawisze powtarzane", -1, 6, 11, 234, 106
     LTEXT "Zastąp ustawienia Klawiatury w Panelu Sterowania:",
           -1, 12, 20, 222, 20
-    AUTORADIOBUTTON "&Bez powtarzania klawiszy", IDC_REPEAT_NOREPEAT_RADIO, 12, 32, 78, 11, WS_GROUP
+    AUTORADIOBUTTON "&Bez powtarzania klawiszy", IDC_REPEAT_NOREPEAT_RADIO, 12, 32, 98, 11, WS_GROUP
```

Comment:
Are these property changes related?

Reply:
These values are enlarged to make the text fully readable.